inst/include/RQuantLib_RcppExports.h

```diff
@@ -385,6 +385,25 @@ namespace RQuantLib {
         return Rcpp::as<QuantLib::Date >(__result);
     }
 
```

Comment:
if you add `// [[Rcpp::export]]` here and re-run `compileAttributes()` the R interface will be created for you.